packages/universal-components/src/Button/ButtonInner.js

```diff
@@ -78,24 +81,40 @@ export default function ButtonInner({
         circled && styleSheet.buttonCircled,
         sizeTheme(size, leftIcon != null, rightIcon != null).buttonSizeWrapper,
         style,
+        isLoading && styleSheet.loadingState,
       ]}
       testID={testID}
     >
-      <View style={layout.row}>
-        {leftIcon != null && <View style={rightSpace}>{leftIcon}</View>}
-        {children ||
-          (label != null && (
-            <ButtonTitle text={label} type={type} size={size} />
-          ))}
-      </View>
-      <View style={layout.row}>
-        {sublabel != null && (
+      {isLoading ? (
+        <View style={{ alignSelf: 'center' }}>
```

Comment:
Please create as style with StyleSheet.create

Reply:
Sorry, forgot to remove it, its not doing anything

Reply:
Removed